docs/source/notebooks.md

```diff
@@ -1 +0,0 @@
-../../notebooks/README.md
```

Comment:
When adding a notebook or any other change we don't want to add the same modification each time to both https://github.com/huggingface/optimum/blob/main/docs/source/notebooks.md and https://github.com/huggingface/optimum/blob/main/notebooks/README.md, would revert